### ISSUE TITLE
Settings

### ISSUE THREAD

**Issue by nakst** [nakst/imgview#3]
Auto-reload image mode.
Remember last window state (maximised/normal).

**Comment by Zero3K** [comment]
I agree with adding those settings. I have two that I would like to be added. The first one is the ability to choose to assign it to image files (with the ability to associate all or specific image extensions). The second is the ability to hide/show the Status Bar.

**Comment by nakst** [comment]
With 509cca7 you can toggle the status bar using `Ctrl+B`.

**Comment by Romz24** [comment]
Do you still plan to implement window position saving? This setting is greatly missed.
